apps/condo/domains/ticket/constants.js

```diff
@@ -55,6 +55,25 @@ const MAX_COMMENT_LENGTH = 700
 
 const DEFAULT_DEFERRED_DAYS = 30
 
+const TICKET_OTHER_SOURCE_ID = '7da1e3be-06ba-4c9e-bba6-f97f278ac6e4'
```

Comment:
Why do we need this const?

Reply:
In tests, I need to check operations for bulk ticket creation. There is no function for creating bulk tickets in test utilities. And I did not write it, because we decided not to support bulk ticket creation. And I need this constant only for creating a ticket that cannot be created

Reply:
It's better to leave it in the test file, or just generate a random id.